pkg/pillar/build-rstats.yml

```diff
@@ -0,0 +1,30 @@
+# linuxkit build template 
+#
+# Copyright (c) 2018 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this file should be kept consistence with build.yml
+org: lfedge
+image: eve-pillar
+config:
+  binds:
+    - /lib/modules:/lib/modules
+    - /dev:/dev
+    - /etc/resolv.conf:/etc/resolv.conf
+    - /run:/run
+    - /config:/config
+    - /:/hostfs
+    - /persist:/persist:rshared,rbind
+    - /usr/bin/containerd:/usr/bin/containerd
+  net: host
```

Comment:
Do you need networking, or can you use ADD in the Dockerfile for all the downloads?

Reply:
We try very hard to use ADD because it makes it easier for SBOM tools to determine exactly what source code versions is included.

Reply:
removed net.